test/user-test.js

```diff
@@ -101,18 +110,42 @@ describe('User', function() {
     expect(user1.favoriteRecipes[0].name).to.equal('Chicken Parm');
   });
 
+  it('should return an alert message if a recipe is undefined', function() {
```

Comment:
Did we get this to send an alert, or return the result? We were having trouble with how to test an alert if I remember right. Maybe the should statement needs changed?

Reply:
That's a good point. I forgot that we had issues with the alert function. We might need to change the test or fix our function